### DISCUSSION TITLE
Using large-v2 with FunASR

### DISCUSSION THREAD

**Discussion by dgoryeo** [openai/whisper#2147] [Q&A] [answered]
I’m exploring the possibility of using the Whisper large-v2 model with the FunASR toolkit for a project. [Alibaba-FunASR](https://github.com/alibaba-damo-academy/FunASR).

Though the FunASR claims that they support Whisper, I cannot find any sample code or any documentation for it.

Would anyone have any insight into that? Any pointers are helpful. Thanks

**Comment by glangford** [comment, marked as answer]
There is this demo/sample code:

https://github.com/alibaba-damo-academy/FunASR/blob/main/examples/industrial_data_pretraining/whisper/demo.py

**Reply by dgoryeo** [reply]
Wow, thanks @glangford !
